### ISSUE TITLE
ERROR: Reverting all changes

### ISSUE THREAD

**Issue by usersina** [wknapik/openvpn-unroot#13]
Running on a fresh install of `Linux version 5.15.47-1-lts (linux-lts@archlinux) (gcc (GCC) 12.1.0, GNU ld (GNU Binutils) 2.38)` gives the following error:

```
INFO: Adding user nobody to group nobody
INFO: Generating sudoers file /etc/sudoers.d/cyberghost-unrooted
INFO: Generating iproute file /etc/openvpn/client/ip-unrooted.sh
INFO: Adding device tun0-unrooted
./openvpn-unroot: line 397: `testing && echo "${comm[@]}" || "${comm[@]}" >/dev/null' returned 1
ERROR: Reverting all changes
./openvpn-unroot: line 499: `local ret; ret=("$("$@")"); readonly ret' returned 1
ERROR: Reverting all changes
```

**Comment by potaesm** [comment]
@usersina Are you able to solve it? I got the same issue on Debian 11.

**Comment by usersina** [comment]
@potaesm nope. I simply enabled the VPN as a [systemd service](https://wiki.archlinux.org/title/OpenVPN#Starting_OpenVPN) and got done with it. That said, internet goes down after a `system suspend` so I have to restart the service every time after I suspend the system.

**Comment by potaesm** [comment]
@usersina Thank you for your reply! That means you are not using anything about openvpn-unroot but just run it as systemd service right?

**Comment by usersina** [comment]
@potaesm That is so. If I ever decide to re-use it and find a fix. I'll post it here!
